### PR TITLE
fix succeed date_format and test

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -370,7 +370,6 @@
     'south',
     'custom.apps.crs_reports',
     'custom.m4change',
-    'custom.succeed'
```
Comment (dannyroberts) [inline settings.py]: there was also never a comma here...

**Comment by benrudolph** [comment]
lol

File: settings.py (hunk 1 above)
Comment (czue) [inline settings.py]: wtf

**Comment by czue** [comment]
build fail doesn't look related, but kicked it just to be sure.

**Comment by czue** [comment]
failed again:
```
......F
======================================================================
FAIL: testExportFilter (corehq.apps.reports.tests.test_export_api.ExportTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_export_api.py", line 124, in testExportFilter
    self.assertEqual(200, resp.status_code)
AssertionError: 200 != 302
----------------------------------------------------------------------
Ran 294 tests in 668.756s
```

**Comment by dannyroberts** [comment]
Yeah i took a quick look yesterday and couldn't figure it out, will take another look shortly

**Pull request merged**